samples/language/DemoProject/src/Main.qs

```diff
@@ -0,0 +1,27 @@
+/// # Sample
+/// Multi File Project
+///
+/// # Description
+/// Organizing code into multiple Q# source files is an important part of
+/// writing readable and maintainable code. In this project, we have `Main.qs`,
+/// and `Particle.qs`, which defines a new namespace for particle operations.
+/// The presence of a Q# manifest file (`qsharp.json`) tells the compiler
+/// to include all Q# files under `src/`.
+namespace MyQuantumApp {
+    open Particle;
+    @EntryPoint()
+    operation Main() : Unit {
```

Comment:
This sample triggers the "operation should be a function" lint, so probably worth making it `function` to avoid confusion.